web/lib/ozoneframework/php/Template/Services/Autoload/LinkService.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace Ozone\Framework\Template\Services\Autoload;
+
+
+
+use Ozone\Framework\TemplateService;
+use Wikidot\Utils\GlobalProperties;
+
+/**
+ * Link-building service.
+ */
+class LinkService extends TemplateService{
+
+	protected $serviceName = "link";
+
+	private $protocol = "http";
```

Comment:
Should this take after `wikijump.ini`s http/https config field?

I see `setSecure()` later, but wouldn't HTTPS be the default in general (if set)?

Reply:
You know, I'm not sure why this was an add in git, this was just a renamed folder.

In any event, there's a `setProtocol($protocol)` method a little further down that allows for https. I think we need a little better understanding of what this does before having it rely on `$HTTP_SCHEMA`.